### PR TITLE
Display selected gyro header value

### LINKED ISSUE #479: Display selected gyro in log header viewer
When doing comparisons between logs on FCs that have two gyros it would be useful to have the gyro selected gyro (first, second, both) saved in the log header.

This goes with https://github.com/betaflight/betaflight/issues/10314 requesting saving selected gyro in blackbox logs

### PR DESCRIPTION
**Pull request by atomgomba**
Fixes #479

### CODE REVIEW

**Review by mikeller — DISMISSED** [review]

**Comment by atomgomba** [comment]
This is a prepared log file I used to test this

[test-headers.bbl.zip](https://github.com/betaflight/blackbox-log-viewer/files/5460214/test-headers.bbl.zip)

**Comment by atomgomba** [comment]
Rebased onto master

File: js/flightlog_fielddefs.js [hunk 1]
```diff
@@ -451,6 +451,12 @@ var
         "KISS",
         "ACTUAL",
         "QUICK",
+    ]),
+
```
Comment (McGiverGim) [inline js/flightlog_fielddefs.js:455]: Sonar complains about trailing spaces here...

**Review by McGiverGim — CHANGES_REQUESTED** [review]

File: js/flightlog_fielddefs.js (hunk 1 above)
Comment (atomgomba) [reply js/flightlog_fielddefs.js]: awh, just vscode things. fixed...

**Review by McGiverGim — APPROVED** [review]
Approved. As suggestion for the next time, is good to paste a screen capture of the change. It helps with the review.

**Review by mikeller — APPROVED** [review]

**Pull request merged**